lib/core/src/sync/model/data.rs

```diff
@@ -37,6 +37,28 @@ impl ChainSyncData {
             }
         }
     }
+
+    pub(crate) fn updated_fields(
+        swap: Option<ChainSwap>,
+        update: &ChainSwap,
+    ) -> Option<Vec<String>> {
+        match swap {
+            Some(swap) => {
+                let mut updated_fields = vec![];
+                if update.accept_zero_conf != swap.accept_zero_conf {
+                    updated_fields.push("accept_zero_conf".to_string());
+                }
+                if update.payer_amount_sat != swap.payer_amount_sat {
+                    updated_fields.push("payer_amount_sat".to_string());
+                }
+                if update.receiver_amount_sat != swap.receiver_amount_sat {
+                    updated_fields.push("receiver_amount_sat".to_string());
+                }
```

Comment:
Does this assume the need to update these fields due to amountless swaps or, more generally, over/underpayments? If not, why would they need to be synced?

Reply:
Yes, I believe it's related to the amountless swaps. Previously those were static fields. @dangeross Do you agree?

Reply:
Yes and I think also now (with Daniel new PR for amountless swaps) they are still static. @danielgranhao is that accurate?

Reply:
Precisely, the changes proposed in #615 make them static again.